spooky-oysters/React-FrontEnd
# Dr post new training program

```sh
git fetch --all
git checkout dr-postNewTrainingProgram
npm start
```
# Steps to Test
copy and paste values into form:
- Program Name: test400
- Start Date: 3/20/2018
- End Date: 3/21/2018
- Max Attendees: 10

Open DB Browser and check that the newest submission exists
